data/actionsmap/yunohost.yml

```diff
@@ -578,6 +578,19 @@ app:
                     full: --app
                     help: Write domain & path to app settings for further checks
 
+        ### app_bookurl()
+        bookurl:
+            action_help: Book a web path for a given app
+            api: GET /tools/bookurl
```

Comment:
Should probably be a POST/PUT request to create a new entry according to Rest semantic.